### PR TITLE
[SCRIPT] - Add static BERT base export script (for use with MXNet Module API)

### PR DESCRIPTION
**Pull request by gigasquid**
## Description ##
This will export the base BERT model for use with the MXNet Module API.

It was adapted from the `static_export_squad.py`

Use cases can include fine tuning for Clojure and Scala APIs. 
* Have used it successfully to reproduce https://gluon-nlp.mxnet.io/examples/sentence_embedding/bert.html in Clojure - PR in MXNet will be coming shortly

### Essentials ###
- [X] PR's title starts with a category (e.g. [BUGFIX], [MODEL], [TUTORIAL], [FEATURE], [DOC], etc)
- [X] Changes are complete (i.e. I finished coding on this PR)
- [ ] All changes have test coverage
- [ ] Code is well-documented

## Comments ##

Python is not my usual language - so feedback is welcome :)

### CODE REVIEW

**Comment by gigasquid** [comment]
Nice bot 🤖

**Comment by mli** [comment]
Job PR-672/1 is complete. 
Docs are uploaded to http://gluon-nlp-staging.s3-accelerate.dualstack.amazonaws.com/PR-672/1/index.html

**Comment by gigasquid** [comment]
@szha The intent is to use of BERT to more than just inference in the Clojure/Scala MXNet apis. Initially, I'm interested in the fine tuning tasks like classification, but feature extraction would be cool too. Basically everything in this http://jalammar.github.io/illustrated-bert/ 

Thanks so much for the static export feature! It opens up a whole new BERT world for the JVM MXNet langs 💯 

@haven-jeon - Good point on the parameters. I'll double check to see if they are all used and remove the ones that are not.

**Comment by gigasquid** [comment]
I was able to put together a walkthrough Clojure jupyter notebook and then export it to markdown 😸 
https://github.com/gigasquid/incubator-mxnet/blob/new-bert-example-with-finetuning/contrib/clojure-package/examples/bert/fine-tune-bert.md

The WIP PR for MXNet is here https://github.com/apache/incubator-mxnet/pull/14769

I'll plan on spending some time tomorrow to double check this script and the params.

**Comment by gigasquid** [comment]
@haven-jeon - I cleaned up some unneeded args and checked the rest. 
Tested with `python static_export_base.py --seq_length 128`

Please take another look when you have a chance.

**Comment by mli** [comment]
Job PR-672/5 is complete. 
Docs are uploaded to http://gluon-nlp-staging.s3-accelerate.dualstack.amazonaws.com/PR-672/5/index.html

File: scripts/bert/staticbert/static_export_base.py [hunk 1]
```diff
@@ -0,0 +1,198 @@
+"""
+Export Base Static Model (BERT)
+
+=========================================================================================
+
+This will export the base BERT model to a static model suitable for use in MXNet Module API.
```
Comment (eric-haibin-lin) [inline scripts/bert/staticbert/static_export_base.py]: Thanks for your contribution! Would you mind adding some documentation to `index.rst`, and a test case in `https://github.com/dmlc/gluon-nlp/blob/master/scripts/tests/test_scripts.py`?

**Comment by szha** [comment]
@gigasquid we recently upgraded the CI setup. It will work once you rebase the PR to latest master.

File: scripts/bert/staticbert/static_export_base.py (hunk 1 above)
Comment (gigasquid) [reply scripts/bert/staticbert/static_export_base.py]: Sure - no problem. I'll do that shortly.

**Comment by mli** [comment]
Job PR-672/1 is complete. 
Docs are uploaded to http://gluon-nlp-staging.s3-accelerate.dualstack.amazonaws.com/PR-672/1/index.html

**Comment by mli** [comment]
Job PR-672/3 is complete. 
Docs are uploaded to http://gluon-nlp-staging.s3-accelerate.dualstack.amazonaws.com/PR-672/3/index.html

**Comment by mli** [comment]
Job PR-672/4 is complete. 
Docs are uploaded to http://gluon-nlp-staging.s3-accelerate.dualstack.amazonaws.com/PR-672/4/index.html

**Comment by mli** [comment]
Job PR-672/5 is complete. 
Docs are uploaded to http://gluon-nlp-staging.s3-accelerate.dualstack.amazonaws.com/PR-672/5/index.html

**Comment by mli** [comment]
Job PR-672/7 is complete. 
Docs are uploaded to http://gluon-nlp-staging.s3-accelerate.dualstack.amazonaws.com/PR-672/7/index.html

**Comment by mli** [comment]
Job PR-672/8 is complete. 
Docs are uploaded to http://gluon-nlp-staging.s3-accelerate.dualstack.amazonaws.com/PR-672/8/index.html

**Comment by mli** [comment]
Job PR-672/10 is complete. 
Docs are uploaded to http://gluon-nlp-staging.s3-accelerate.dualstack.amazonaws.com/PR-672/10/index.html

**Comment by gigasquid** [comment]
tests & docs adds and CI is green 💚 - please take another look when you get a chance

**Pull request merged**